src/app/components/profile/profile.component.html

```diff
@@ -1,76 +1,126 @@
+						<div class="col s12 m4 l4 center user-info" id="user_info">
+							<div class="user-details">
+								<ul>
+									<li>
+								<div class="item">
+									<span class="text-light-black fw-semibold fs-12">
+										First Name<br>
+									</span>
+									<div class="content fw-regular">
+										{{user.first_name}}
+									</div>
+								</div>
+									</li>
+									<li>
+								<div class="item">
+									<span class="text-light-black fw-semibold fs-12">
+										Last Name<br>
+									</span>
+									<div class="content fw-regular">
+										{{user.last_name}}
+									</div>
+								</div>
+								</li>
+								<li>
+								<div class="item">
+									<span class="text-light-black fw-semibold fs-12">
+										Affiliated to<br>
+									</span>
+									<div class="content fw-regular">
+										{{user.affiliation}}
+									</div>
+								</div>
+								</li>
+								</ul>
+								</div>
+							</div>		
+							<div class="col s12 m5 l5 user-urls center" >
+								<ul>
+									<li>
+									<div class="item">
+										<span class="text-light-black fw-semibold fs-12">
+										GitHub Url<br>
+										</span>
+									<div class="content fw-regular">
+										{{user.github_url}}
+									</div>
+									</div>
+								</li>
+								<li>
+							    <div class="item">
+								<span class="text-light-black fw-semibold fs-12">
+								Google Scholar Url<br>
+								</span>
+								<div class="content fw-regular">
+								{{user.google_scholar_url}}
+								</div>
+							    </div>
+								</li>
+								<li>
+								<div class="item">
+									<span class="text-light-black fw-semibold fs-12">
+										Linkedin Url<br>
+									</span>
+									<div class="content fw-regular">
+										{{user.linkedin_url}}
+									</div>
+								</div>
+							</li>
+						</ul>
+						<ul class="inline-list profile-edit-group center" id="sty_button">
+							<li >
+								<a (click)="updatePassword()" class="dark-link pointer">
+									<strong class="fw-semibold fs-15 change-password-text">Change Password</strong>
+								</a>
+							</li>
+							<li>
+								<button (click)="updateUserDetails()" class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14">Update Profile</button>
+							</li>
+						</ul>
+					</div>
+					<div class="modal-container" [class.hidden]="!isTokenModalVisible">
+						<div class="modal-card">
+							<div class="title">
+								API Authentication Token
+							</div>
+							<div class="form row">
+								<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
+									<app-input [label]="'token'" [placeholder]="'Auth Token'"
 									[type]="'password'" [theme]="'dark'" [value]="token" [readonly]="true"
 									#formtoken></app-input>
-							</div>
-							<button class="col-lg-1 col-md-1 col-sm-1 col-xs-1 btn ev-btn-dark
-								btn-waves-effect waves-dark grad-btn code-btn"
+								</div>
```

Comment:
The whitespace is not necessary i think.

Reply:
i cant understand?